db/view/view.cc

```diff
@@ -2646,7 +2646,7 @@ update_backlog node_update_backlog::fetch() {
         _max.store(new_max, std::memory_order_relaxed);
         return new_max;
     }
-    return std::max(fetch_shard(this_shard_id()), _max.load(std::memory_order_relaxed));
+    return _max.load(std::memory_order_relaxed);
```

Comment:
I wonder, wouldn't it be better to keep it? In case when there is a shard-aware write, we will get quite an up-to-date backlog. Otherwise, we will get a backlog which might only be higher due to the `max` with the local shard's backlog, but it's not wrong to do so. WDUT?

Reply:
Makes sense, I reverted the deletion in a rebase, but to avoid future confusion about where we expected the backlog to have been updated, I added a comment that it's just likely in a shard-aware scenario